php/php.latte/test/unit/data/testfiles/indent/testRealFile_01.latte

```diff
@@ -0,0 +1,212 @@
+{*
+ * File source: https://github.com/obrejla/Eciovni/blob/master/src/Eciovni.latte
```

Comment:
@matthiasblaesing I think that we have to write the correct license. Am I wrong?

Reply:
Here is text of license fot source package: https://github.com/obrejla/Eciovni/blob/master/license.txt
So I'll put it in comment at the begining of the file, right?

Reply:
Maybe, Yes. Matthias should tell us the proper way.

Reply:
I found PRs #751 and #720 adding external files with New BSD License.
I prepared added files in same way.

Reply:
@KacerCZ No. They were added as external files because the license is GPL(we can't add files of GPL). Anyway, please wait until you get a reply.
NOTE: You might not be able to get a response, review, and more soon, but just wait...

Reply:
@junichi11 The mentioned PRs also added files in similar case as discussed here (third party code, New BDS License):
- https://github.com/apache/netbeans/blob/master/php/php.editor/test/unit/data/testfiles/actions/testImportData/libs/nette.min.php
- https://github.com/apache/netbeans/blob/master/php/php.editor/test/unit/data/testfiles/verification/testEmptyStatementHint.php
I believe my last changes made this compliant with licensing requirements. Now I'll wait for review.